Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -24,26 +32,31 @@ class LayoutManager: NSLayoutManager {
                     return
             }
 
-            let borderColor = UIColor(red: 0.5294117647, green: 0.650980392156863, blue: 0.737254902, alpha: 1.0)
-            let backgroundColor = UIColor(red: 0.91, green: 0.94, blue: 0.95, alpha: 1.0)
             let blockquoteGlyphRange = glyphRange(forCharacterRange: range, actualCharacterRange: nil)
 
             enumerateLineFragments(forGlyphRange: blockquoteGlyphRange) { (rect, usedRect, textContainer, glyphRange, stop) in
                 let lineRect = rect.offsetBy(dx: origin.x, dy: origin.y)
-                backgroundColor.setFill()
+                self.blockquoteBackgroundColor.setFill()
                 context.fill(lineRect)
                 let borderRect = CGRect(origin: lineRect.origin, size: CGSize(width: 2, height: lineRect.height))
-                borderColor.setFill()
+                self.blockquoteBorderColor.setFill()
                 context.fill(borderRect)
             }
         }
+    }
+
+    private func drawLists(forGlyphRange glyphsToShow: NSRange, at origin: CGPoint) {
+        guard let textStorage = textStorage else {
+            return
+        }
 
+        let characterRange = self.characterRange(forGlyphRange: glyphsToShow, actualGlyphRange: nil)
         // draw list markers
         textStorage.enumerateAttribute(NSParagraphStyleAttributeName, in: characterRange, options: []){ (object, range, stop) in
             guard let paragraphStyle = object as? ParagraphStyle,
-                  let _ = paragraphStyle.textList
+                let _ = paragraphStyle.textList
```

Comment:
Note: You can leave this as it is right now if you prefer.

IMHO whenever we're not using the value checking for `paragraphStyle.textList != nil` is somewhat clearer.